cli/test/test_text2embeddings.py

```diff
@@ -0,0 +1,75 @@
+from pathlib import Path
+import tempfile
+from unittest import mock
+
+from click.testing import CliRunner
+from cloudpathlib.local import LocalS3Path
+import pytest
+import numpy as np
+
+from cli.text2embeddings import main as cli_main
+
+
+@pytest.fixture()
+def test_input_dir() -> Path:
+    return (Path(__file__).parent / "test_data" / "text2embeddings_input").resolve()
+
+
+def test_run_encoder_local(test_input_dir: Path):
+    """Test that the encoder runs with local input and output paths and outputs the correct files."""
+
+    with tempfile.TemporaryDirectory() as output_dir:
+        runner = CliRunner()
+        result = runner.invoke(cli_main, [str(test_input_dir), output_dir])
+        assert result.exit_code == 0
+
+        assert (Path(output_dir) / "test_html.npy").exists()
+        assert (Path(output_dir) / "test_html.json").exists()
+        assert (Path(output_dir) / "test_pdf.npy").exists()
+        assert (Path(output_dir) / "test_pdf.json").exists()
```

Comment:
Is there a test that validates this is in fact json and/or with the correct schema?